config/webpack.config.js

```diff
@@ -34,7 +34,7 @@ const configGenerator = (options) => {
     filesToBuild = _.pick(entryFiles, options.entry.split(',').map(x => x.trim()));
   }
   filesToBuild.vendor = [
-    'core-js',
+    './src/js/common/polyfills',
```

Comment:
I think perhaps I'm misunderstanding what this `filesToBuild.vendor` is for. Does it include these in the vendor bundle, or does it just transpile these (or some such)?

If it includes them, do we need to `require('./polyfills')` from `src/js/common/index.js`, or does that line just pull them into working memory from the vendor bundle where they reside?